### DISCUSSION TITLE
How to retrieve an item and its relationships?

### DISCUSSION THREAD

**Discussion by fusigabs** [directus/directus#2851] [Archive]
Is there any way to retrieve an item and it's related records? eg, how to retrieve `Categories` together with all `Posts` in each category? Maybe an `include` in the query to eagerload records?

**Comment by rijkvanzanten** [comment]
Use the `fields` parameter: https://docs.directus.io/api/query/fields.html

**Reply by fusigabs** [reply]
Great, thanks.
